frontend/src/components/maps/leaflet/MapProperties.test.tsx

```diff
@@ -108,8 +108,8 @@ describe('MapProperties View', () => {
     expect(tree).toMatchSnapshot();
   });
 
-  // Check that the markers have correct position given the mock parcel 1 above
-  it('Renders the marker in correct position', async () => {
+  // TODO: Check that the markers have correct position given the mock parcel 1 above
+  xit('Renders the marker in correct position', async () => {
```

Comment:
As mentioned in the PR notes these tests fail because they rely on how the map pins are drawn on the map. Clustering has changed this dramatically so we need to re-do these tests (in a future PR)